ExamenProeveUnity/Assets/Scripts/Runtime/Grabbable.cs

```diff
@@ -26,6 +28,10 @@ public class Grabbable : MonoBehaviour, IGrabbable
         private void Awake()
         {
             _isInteractable = gameObject.HasComponent<Interactable>();
+            if (icon == null)
+            {
+                icon = gameObject.HasComponent<Product>() ? gameObject.GetComponent<Product>().productScriptableObject.Icon : null;
+            }
```

Comment:
trouwens ook een rare connection waarom zou grabable de product class kennen (ja je kan nu makkelijk products asignen maar connection wise is het beetje raar maar je mag het houden)